### PR TITLE
feat(ui): Host platform images on datahub-web-react

### PR DESCRIPTION
**Pull request by ngamanda**
Adds capability for users to host their platform images (from `src/images`) on their react app instead.

Context: https://datahubspace.slack.com/archives/CV2UXSE9L/p1644477881072309

## Changes
- Installs [CopyWebpackPlugin](https://webpack.js.org/plugins/copy-webpack-plugin/) to copy `src/images` to `platforms`
- [Updated] `data_platforms.json` files are updated to use the new relative url

## Sample Usage:
```jsonc
// data_platforms.json
{
    "urn": "urn:li:dataPlatform:hive",
    "aspect": {
      "datasetNameDelimiter": ".",
      "name": "hive",
      "displayName": "Hive",
      "type": "FILE_SYSTEM",
      - "logoUrl": "https://raw.githubusercontent.com/linkedin/datahub/master/datahub-web-react/src/images/hivelogo.png"
      + "logoUrl": "/assets/platforms/hivelogo.png"
    }
  },
``` 

## Checklist
- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [x] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

**Review by gabe-lyons — COMMENTED** [review]
Hey @ngamanda, thoughts on just doing this for everyone? I think most (if not all) of folks would prefer hosting their own platform logos, and the additional cost should be pretty small. Any reason not to automatically do this?

File: datahub-web-react/craco.config.js [hunk 1]
```diff
@@ -13,6 +14,15 @@ function addLessPrefixToKeys(styles) {
...
 module.exports = {
+    webpack: {
+        plugins: {
+            add: [
+                new CopyWebpackPlugin({
+                    patterns: [{ from: 'src/images', to: 'platforms' }],
+                }),
+            ],
+        },
+    },
```
Comment (gabe-lyons) [inline datahub-web-react/craco.config.js:25]: can we just add an explanation for why the copy is needed? otherwise looks great!! thanks @ngamanda

**Review by gabe-lyons — APPROVED** [review]

File: datahub-web-react/craco.config.js (hunk 1 above)
Comment (ngamanda) [reply datahub-web-react/craco.config.js]: Added!

**Review by shirshanka — APPROVED** [review]
LGTM!

**Pull request merged**